teams/vsp/teams/triage/README.md

```diff
@@ -1,45 +1,37 @@
 # Triage
 
+Repository for team information, planning docs, meeting notes, etc.
+
 ### Mission
-Identify, resolve, and delegate issues across the platform and applications for Veterans and teams building products to serve Veterans while continuously improving telemetry, tools, and documentation.
+Identify, resolve, and delegate issues across the platform and applications for Veterans and teams building products to serve Veterans while 
```

Comment:
Did you mean to delete "continuously improving telemetry, tools, and documentation." from the sentence here?

Reply:
Nope. Fixed.


I *do* think we need to revisit this mission and clarify the language. It's not very clear to understand.